src/mito2/src/worker/handle_alter.rs

```diff
@@ -167,6 +169,67 @@ impl<S> RegionWorkerLoop<S> {
                         current_options.ttl = Some(new_ttl);
                     }
                 }
+                ChangeOption::TwscMaxActiveWindowRuns(runs) => {
```

Comment:
If we change all these variants into `Twcs(String)`, then we can refactor these match clauses into:

```rust
ChangeOption::Twcs(options) =>
{
    match options.as_str() => {
       "compaction.twcs.max_active_window_runs" => { .. }
      ......
    }
}
```